sunpy/net/dataretriever/tests/test_eve.py

```diff
@@ -84,7 +85,11 @@ def test_fido(query):
 
 
 @pytest.mark.remote_data
-@given(time_attr(time=datetimes(timezones=[], max_year=datetime.datetime.utcnow().year, min_year=2010)))
+@given(time_attr(time=datetimes(
+    # timezones=None,
```

Comment:
Why is this commented out? is it needed?

Reply:
No it is not needed. I'll remove it in next commit.